oliviacweb/CopWatch-FE
# Feature/report form

#### What's this PR do?

* Adds ReportForm with basic inputs for the report. Also, all tests are passing or skipped to test Travis CI.


#### Where should the reviewer start?

* ReportForm.js


#### How should this be manually tested?

* Enter input in forms and check to see if hooks were update with react dev tools.


#### Any background context you want to provide?

*  n/a


#### What are the relevant tickets?

* #27
